launcher/ui/pages/modplatform/ResourcePage.ui

```diff
@@ -11,35 +11,55 @@
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
-   <item row="1" column="0" colspan="4">
-    <layout class="QGridLayout" name="gridLayout_3">
-     <item row="1" column="2">
-      <widget class="ProjectDescriptionPage" name="packDescription">
-       <property name="openExternalLinks">
-        <bool>false</bool>
-       </property>
-       <property name="openLinks">
-        <bool>false</bool>
-       </property>
-      </widget>
-     </item>
-     <item row="1" column="0">
-      <widget class="QListView" name="packView">
-       <property name="horizontalScrollBarPolicy">
-        <enum>Qt::ScrollBarAlwaysOff</enum>
-       </property>
-       <property name="alternatingRowColors">
-        <bool>true</bool>
-       </property>
-       <property name="iconSize">
-        <size>
-         <width>48</width>
-         <height>48</height>
-        </size>
-       </property>
-      </widget>
-     </item>
-    </layout>
+   <item row="0" column="2">
+    <widget class="Line" name="line">
+     <property name="orientation">
+      <enum>Qt::Vertical</enum>
+     </property>
+    </widget>
+   </item>
+   <item row="2" column="0" colspan="4">
+    <widget class="ProgressWidget" name="fetchProgress" native="true"/>
+   </item>
+   <item row="0" column="1">
+    <widget class="QPushButton" name="resourceFilterButton">
+     <property name="text">
+      <string>Filter options</string>
```

Comment:
could this button be on the left of the search bar so it is closer to the filter panel maybe?

Reply:
Yes I can